src/main/routes/routes.ts

```diff
@@ -83,6 +83,11 @@ export default function(app: Application): void {
   app.post('/subscription-add', ensureAuthenticated, app.locals.container.cradle.subscriptionAddController.post);
   app.get('/case-event-glossary', app.locals.container.cradle.caseEventGlossaryController.get);
 
+  app.get('/case-reference-number-search', app.locals.container.cradle.caseReferenceNumberSearchController.get);
```

Comment:
youve not protected these under authenticated path

Reply:
these has been changed to protected.